src/esp/scene/GibsonSemanticScene.cpp

```diff
@@ -0,0 +1,75 @@
+// Copyright (c) Facebook, Inc. and its affiliates.
+// This source code is licensed under the MIT license found in the
+// LICENSE file in the root directory of this source tree.
+
+#include "GibsonSemanticScene.h"
+#include "SemanticScene.h"
+
+#include <map>
+#include <string>
+
+#include <Corrade/Utility/String.h>
+
+#include "esp/io/io.h"
+#include "esp/io/json.h"
+
+namespace esp {
+namespace scene {
+
+constexpr int kMaxIds = 10000; /* We shouldn't every need more than this. */
+
+bool SemanticScene::loadGibsonHouse(
+    const std::string& houseFilename,
+    SemanticScene& scene,
+    const quatf& worldRotation /* = quatf::Identity() */) {
+  if (!io::exists(houseFilename)) {
```

Comment:
`Cr::Utility::Directory::exists()`? IIRC the `io::` APIs are obsolete and should be gradually replaced with corrade's functionality.